### DISCUSSION TITLE
Hazardous specimens like arsenic etc. in Arctos

### DISCUSSION THREAD

**Discussion by lin-fred** [ArctosDB/arctos#5555] [Announcements]
Hey all I have several questions about hazard specimens, in my case it is specifically arsenic, but any insight is welcome.

What do you all do with objects/specimens in Arctos to specify they have an issue like this? Just in part remarks or?

How do you all confirm something has Arsenic as well. As in, you suspect it has arsenic, like this one: https://arctos.database.museum/guid/NMMNH:Bird:298

Then, if you are pretty sure/are sure that it has arsenic (or something else), do you put it on exhibit. And if so how? And if not, why?

Thanks all!

**Comment by lin-fred** [comment]
I've noticed that there is an option for this information in preservation, but does anyone add it to a project or something so they're easy to find?

**Comment by Jegelewicz** [comment]
One easy thing would be a saved search - it will automatically update is new information is added.

**Comment by AJLinn** [comment]
You could also create a project for each hazardous material you’re tracking, create a loan to add objects to each year, and then relate that loan to the project each year.

Our colleagues at UAMN are using portable XRF machines to detect arsenic and other hazardous materials in natural history specimens. I think it’s been especially prevalent in our bird collections, but has also shown up on taxidermied specimens in galleries. I know I’ve got an article about this… probably from SPNHC. I’ll see if I can find it and attach/link.

**Comment by AJLinn** [comment]
Here’s an MA thesis on the topic: [Poisonous Heritage](https://scholarship.shu.edu/cgi/viewcontent.cgi?article=1255&=&context=theses&=&sei-redir=1&referer=https%253A%252F%252Fscholar.google.com%252Fscholar%253Fhl%253Den%2526as_sdt%253D0%25252C2%2526q%253Dpesticides%252Bin%252Bmounted%252Bspecimens%2526btnG%253D#search=%22pesticides%20mounted%20specimens%22): Pesticides in Museum Collections

here’s the whole issue of [Collection Forum](https://www.researchgate.net/profile/Catharine-Hawks-2/publication/237561156_Exposure_of_museum_staff_to_formaldehyde_during_some_wet_specimen_activities/links/53e511ea0cf2fb7487124025/Exposure-of-museum-staff-to-formaldehyde-during-some-wet-specimen-activities.pdf#page=53) on pesticides in collections!

**Comment by Jegelewicz** [comment]
Yeah - the loan/project path is really the best way....

**Comment by ewommack** [comment]
We created a collections wide Arsenic policy and safety training before anyone can touch anything in our collection. Since we didn't have the money to test all of the historic collection, we ramped out a basic: if it was _prepped_ before 2000 and the preparator cannot be reached, assumed that it might have arsenic and it falls under the arsenic protocols.
Happy to share @lin-fred our protocols if that might help.

It might be good to figure this out for other things as well, like objects that were confirmed to have a specific disease in life, e.g. rabies, west nile, etc.

**Comment by AJLinn** [comment]
@ewommack I'd love to see your protocols - we have items in the cultural collections at UAMN that I'm concerned about also - bird skin parkas in particular. I've started thinking about other risks in our collection (live ammunition, darts with curare tips, objects treated with pesticides, etc.) and might be considering a grant proposal to do more work on it here. It would be good to think about what tools we have for flagging hazardous items in the collection via Arctos.

**Comment by ewommack** [comment]
@AJLinn - here you go. We worked really closely with our safety office to design these, and figure out something that would keep people as safe as possible while also allowing use of the collections. 
We did basic testing through the collection of specific types of material and did detect arsenic, but didn't run the types of tests that would determine how much. The cost per object was too high for us to do that type of analysis. We also did specific airborne particle safety tests, where we strapped detectors onto employees and tested them through out normal and more extreme operations. It was nice to know that when we turn and fold our entire historic pelt collection we we didn't create big clouds of lead or arsenic dust.

[Non-Arsenic SOP.pdf](https://github.com/ArctosDB/arctos/files/8758202/Non-Arsenic.SOP.pdf)
[Arsenic SOP.pdf](https://github.com/ArctosDB/arctos/files/8758203/Arsenic.SOP.pdf)

I'd highly recommend reaching out to your safety office, and at least opening up a conversation. One of the big things for them was to come down and actually learn about how we use the objects and space. That really helped convince them we had to figure out how to make them still accessible, while also increasing the safety protocols.

**Comment by wellerjes** [comment]
Hi all,

I'd like to bring this topic back to the forefront. We have specimens in the botany collection that were treated with PBD and thymol in the 1940s and I'm not sure where the best place to document this in the catalog record.

I was about to request that these values be added to the [part preservation code table](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation), but I noticed that arsenic was not an option in this code table, which reminded me of this conversation. Is this the appropriate code table to request these values? Or do we need to explore the option of a new code table (or new value under something like "processing history")?

Thanks!

**Reply by dustymc** [reply]
>part preservation code table

[Yup.](https://github.com/ArctosDB/arctos/issues/new?assignees=&labels=Function-CodeTables&projects=&template=code-table-request.md&title=Code+Table+Request+-+)
